include/vrv/horizontalaligner.h

```diff
@@ -319,9 +319,9 @@ class AlignmentReference : public Object, public AttNInteger {
     bool HasMultipleLayer() const { return (m_layerCount > 1); }
 
     /**
-     * Return first accidental from the accid space of the alignment reference
+     * Return const pointer to the accid space
      */
-    Accid *GetFrontAccid() const;
+    const std::vector<Accid *> *GetAccidSpace(bool docChildren = true) const { return &m_accidSpace; }
```

Comment:
What's the purpose of the `docChildren` parameter?

Reply:
Yeah, this should not be there. I'll remove it.